soccer/planning/TargetVelPathPlanner.cpp

```diff
@@ -93,26 +104,28 @@ bool TargetVelPathPlanner::shouldReplan(
 // TODO(justbuchanan): Paths aren't dynamically feasible sometimes because it
 // doesn't account for initial velocity
 std::unique_ptr<Path> TargetVelPathPlanner::run(
-    MotionInstant startInstant, MotionCommand cmd,
+    MotionInstant startInstant, const MotionCommand* cmd,
     const MotionConstraints& motionConstraints,
     const Geometry2d::ShapeSet* obstacles, std::unique_ptr<Path> prevPath) {
-    assert(cmd.getCommandType() == MotionCommand::WorldVel);
-
-    // If the start point is in an obstacle, escape from it
```

Comment:
Yeah but that's a separate thing.  There was a chunk of code here that handled the case where the start point is inside an obstacle.  Looks like you hit a merge conflict and chose your changes when you should have chosen both.